drv/stm32xx-i2c/src/lib.rs

```diff
@@ -400,21 +367,16 @@ impl I2cController<'_> {
             //   t_timeout = (TIMEOUTA + 1) x 2048 x t_i2cclk
             //
             // We want our t_timeout to be at least 25 ms: on h743 with a 10 ns
-            // t_i2cclk this yields 1219.7 (1220); on h7b3, this is 3416.9
-            // (3417); on g031, this is 195.88 (196).
+            // t_i2cclk this yields 1219.7 (1220); on g031, this is 195.88
+            // (196). Note that these numbers make assumptions about the
+            // system's clocking and clock tree configuration; TODO.
```

Comment:
**question:** What is the TODO here about exactly? Is it about finishing this comment or something code related?

Reply:
The TODO is intended to be about the earlier part of the sentence: when visiting this file I noticed that it contains hardcoded assumptions about the system's clocking and clock tree configuration, which are under the control of the application main file. This should eventually be fixed.

Reply:
Thank you, that explains it really well.